app/src/main/java/calculator/example/com/kotlincalculator/mvp/view/ActivityView.kt

```diff
@@ -0,0 +1,13 @@
+package calculator.example.com.kotlincalculator.mvp.view
+
+import android.app.Activity
+
+import java.lang.ref.WeakReference
+
+open class ActivityView(activity: Activity) {
```

Comment:
I'd suggest to make it abstract, so there's no need for explicitly opening it

Reply:
Done 👍